src/transformers/pipelines/automatic_speech_recognition.py

```diff
@@ -208,7 +172,13 @@ def _sanitize_parameters(self, **kwargs):
             preprocess_params["chunk_length_s"] = kwargs["chunk_length_s"]
         if "stride_length_s" in kwargs:
             preprocess_params["stride_length_s"] = kwargs["stride_length_s"]
-        return preprocess_params, {}, {}
+
+        postprocess_params = {}
+        if "raw_ctc" in kwargs:
```

Comment:
It is captured by `postprocess`, however I don't think it's needed really.

Most likely some local experiment that I ended committing by error.
FYI, the goal was to enable the caller to fuse the chunks themselves, in the context of a stateless pipeline that would still be usable by a live microphone. Since the pipeline is stateless, it couldn't know the last used token in order to set it in the striding area (so it would get properly discarded when doing CTC decoding).

But as I found out, a purely stateless pipeline, incurs way to much network overhead to be viable (The current script settings would mean 4Mo/s bandwidth, vs 64ko with a stateful connection, so stateful connection it is, and so we can ignore `raw_ctc` altogether :)).

Reply:
Removed it from current PR, it also removes the other issue.

Reply:
It was used for the 2nd demo btw, where it was live and perfect transcription. But we will settle for the updated second demo which might contain erroneously duplicated letters (due to the chunking boundaries). The demo still looks&feels nice IMO.